src/txn_fuzzer.rs

```diff
@@ -326,7 +326,7 @@ impl From<LoadAndExecuteTransactionsOutput> for TxnResult {
             is_ok,
             status,
             instruction_error,
-            instruction_error_index: instruction_error_index.into(),
+            instruction_error_index: instruction_error_index,
```

Comment:
do `instruction_error_index` instead of `instruction_error_index: instruction_error_index`